hunts/src/AddPuzzleModal.js

```diff
@@ -53,6 +58,25 @@ function AddPuzzleModal({ huntId }) {
               onChange={(e) => setUrl(e.target.value)}
             />
           </Form.Group>
+          <Form.Group controlId="addPuzzle.meta">
+            <Form.Label>Assigned Meta</Form.Label>
+            <Form.Control
+              as="select"
+              value={assignedMeta}
```

Comment:
If you use a falsy value like `null` (@rgossiaux's comment above), you can probably change this to `value={assignedMeta ?? "none")`.

Reply:
I'll just change both to "".